magento/magento2
# Fix selections for bundle options of type checkbox are wrong on the wishlist

## Description (*)
Fixed wrong behavior, when all selections of a bundle option of type checkbox that were checked once are added to the wishlist - no matter if they were unchecked later, before the bundle is added to the wishlist.

### Actual result
The selections of the bundle product on the wishlist match the previously chosen ones, selection that unchecked again are not listed.

### Manual testing scenarios (*)
Create a bundle product with at least one option of type checkbox
Add several (at least three) simple products to this option
Access the bundle product on frontend
Select and again deselect some of the checkbox options' selections
Add the bundle product to your wishlist
Take a look at "See Details" popup.

### Contribution checklist (*)
* [x]   All new or changed code is covered with functional test
